test_rclcpp/test/test_timeout_subscriber.cpp

```diff
@@ -51,11 +51,15 @@ TEST_F(CLASSNAME(test_timeout_subscriber, RMW_IMPLEMENTATION), timeout_subscribe
   auto start = std::chrono::steady_clock::now();
 
   auto node = rclcpp::Node::make_shared("test_timeout_subscriber");
+  auto cg = node->create_callback_group(rclcpp::CallbackGroupType::MutuallyExclusive);
```

Comment:
It might be worth leaving a comment here for posterity. Otherwise, someone might come back and refactor this to the way it was before.

Reply:
what would be the comment?
Really I still don't get the goal of the test :joy:, this change just makes it not fail randomly.

Reply:
Maybe `// Add subscription to its own callback group to avoid interference from other things in the node` ?

Reply:
```suggestion
  // Add subscription to its own callback group to avoid interference from other things in the node
  auto cg = node->create_callback_group(rclcpp::CallbackGroupType::MutuallyExclusive);
```

Reply:
done in https://github.com/ros2/system_tests/pull/465/commits/f095533e21eaf4c7f81d1e9ee9d1e998ad1abef1